### ISSUE TITLE
Odd behavior on nully (leading) values

### ISSUE THREAD

**Issue by gderosa** [touv/node-csv-string#17]
```
var csv = require('csv-string')
var obj = {a:null, b:'', c:'aValue'}
var ary = [null, '', 'aValue'] // the values of obj
```

Now, `csv.stringify(ary)` gives you `',,aValue\r\n'`.

But `csv.stringify(obj)` returns `'aValue\r\n'`, the other two values are just skipped, rather than being rendered as empty cells.

I think this is a bug (as it would break alignment in a csv file)?

Thanks!

**Comment by touv** [comment]
fixed in version 2.2.5

**Comment by gderosa** [comment]
Thank you.
